Update visibility tutorials (#272)
otterize/docs: docs/features/azure-iam/tutorials/azure-iam-aks.mdx

To deploy Otterize, head over to [Otterize Cloud](https://app.otterize.com) and:

1. Create a Kubernetes integration on the [Integrations page](https://app.otterize.com/integrations), and follow the instructions. *Make sure to enable enforcement mode for this tutorial.* If you already have a Kubernetes cluster connected, skip this step.
    - Add the following value to your helm installtion, to enable Azure resource mapping with eBPF:
        ```
        helm upgrade ... --set networkMapper.nodeagent.enable=true

>         ```

Reviewer: This should look like the info callout in the AWS tutorial (the `:::info ....` section)

By annotating the pod, we've created a workload identity.
We now need to specify what we need to access, and the intents operator will create an Azure IAM role assignment accordingly.

To do so, we will apply a `ClientIntents` resource that permits the client pod to download files from the source container:
```bash
kubectl apply -n otterize-tutorial-azure-iam -f ${ABSOLUTE_URL}/code-examples/azure-iam-aks/clientintents-1.yaml
kubectl patch -n otterize-tutorial-azure-iam clientintents/client-intents-for-client  --type='json' -p="[{\"op\": \"replace\", \"path\": \"/spec/calls/0/name\", \"value\": \"/providers/Microsoft.Storage/storageAccounts/$STORAGE_ACCOUNT_NAME/blobServices/default/containers/downloads\"}]"
```

> This applies the following ClientIntents, grant read access to the `downloads` container in the storage account:

Reviewer: ```suggestion
This applies the following ClientIntents, granting read access to the `downloads` container in the storage account:
```

```yaml
{@include: ../../../../static/code-examples/azure-iam-aks/clientintents-1.yaml}
```

> ### The client can now download files in the Azure Blob Storage container!

Reviewer: ```suggestion
### The client can now download files from the Azure Blob Storage container!
```

Depending on your operation, you may need to be assigned one of the following roles:                                                            │
    "Storage Blob Data Owner"                                                                                                                   │
    "Storage Blob Data Contributor"                                                                                                             │
    "Storage Blob Data Reader"                                                                                                                  │
    "Storage Queue Data Contributor"                                                                                                            │
    "Storage Queue Data Reader"                                                                                                                 │
    "Storage Table Data Contributor"                                                                                                            │
    "Storage Table Data Reader"
```

> The file is successfully downloader from the `downloads` container, but we fail to upload it to the `uploads` container. This is because our intents file didn't specify all required permissions.
> 
> We can fix this by taking a look at the access graph at [Otterize Cloud](https://app.otterize.com) and adding the necessary permissions to the `ClientIntents` resource:
> 
> ![Access Graph](/img/quick-tutorials/azure-iam-aks/accessgraph.png)
> 
> This graph shows a green line where intents were applied, and a yellow line where access was discovered, but not yet granted. This is the access we need to add to the `ClientIntents` resource. Click on the yellow line to see the required permissions, and apply them to the cluster.

Reviewer: ```suggestion
The file is successfully downloaded from the `downloads` container, but the client fails to upload it to the `uploads` container. This is because some permissions are still missing. Let's take a look at the access graph at [Otterize Cloud](https://app.otterize.com): 

![Access Graph](/img/quick-tutorials/azure-iam-aks/accessgraph.png)

The access graph visualizes the applied ClientIntents in green, while the newly discovered intents, which are still missing permissions, are shown in yellow. 

By clicking on the `client` node, you can view and download the full set of ClientIntents required for it to access both buckets: 

<add a screenshot here of the access graph with the ClientIntents yaml pane open>

Next, apply the full ClientIntents, which add the required permissions to allow uploading files to the upload bucket:
```

```bash
kubectl apply -n otterize-tutorial-azure-iam -f ${ABSOLUTE_URL}/code-examples/azure-iam-aks/clientintents-2.yaml
kubectl patch -n otterize-tutorial-azure-iam clientintents/client-intents-for-client  --type='json' -p="[{\"op\": \"replace\", \"path\": \"/spec/calls/0/name\", \"value\": \"/providers/Microsoft.Storage/storageAccounts/$STORAGE_ACCOUNT_NAME/blobServices/default/containers/downloads\"}]"

> kubectl patch -n otterize-tutorial-azure-iam clientintents/client-intents-for-client  --type='json' -p="[{\"op\": \"replace\", \"path\": \"/spec/calls/1/name\", \"value\": \"/providers/Microsoft.Storage/storageAccounts/$STORAGE_ACCOUNT_NAME/blobServices/default/containers/downloads\"}]"

Reviewer: you can do this in a single patch (with two ops)

Author: replaced with otterize-cli

    "Storage Queue Data Reader"                                                                                                                 │
    "Storage Table Data Contributor"                                                                                                            │
    "Storage Table Data Reader"
```

The file is successfully downloaded from the `downloads` container, but the client fails to upload it to the `uploads` container. This is because some permissions are still missing. Let's take a look at the access graph at [Otterize Cloud](https://app.otterize.com): 

![Access Graph](/img/quick-tutorials/azure-iam-aks/accessgraph.png)

The access graph visualizes the applied ClientIntents in green, while the newly discovered intents, which are still missing permissions, are shown in yellow. 

By clicking on the `client` node, you can view and download the full set of ClientIntents required for it to access both buckets: 

![Client Intents](/img/quick-tutorials/azure-iam-aks/clientintents.png)

Use the otterize CLI to download and apply the intents to the cluster:

```bash

> otterize clientintents export --api-address https://app.staging.otterize.com/api -n otterize-tutorial-azure-iam.<CLUSTER NAME> -v v2 | kubectl apply -f-

Reviewer 2: fix me and anything similar (reference to staging)
